lib/libzrepl/mgmt_conn.c

```diff
@@ -1104,6 +1281,12 @@ process_message(uzfs_mgmt_conn_t *conn)
 			    hdrp->opcode, hdrp->io_seq);
 			break;
 		}
+		if (hdrp->opcode == ZVOL_OPCODE_SNAP_LIST) {
+			LOGCONN(conn, "Snaplist command for %s", zinfo->name);
+			rc = uzfs_zvol_dispatch_command(conn, hdrp, NULL, 0,
```

Comment:
passing '0' can make uzfs_zvol_dispatch_command to alloc of size 0, and, may have issue. we need to do checks for zero length while allocating memory checks at some point.

Reply:
ok